rest/trades/trades_test.go

```diff
@@ -165,7 +165,7 @@ func TestGetLastCryptoTrade(t *testing.T) {
 	assert.Equal(t, expectedResponse, string(b))
 }
 
-func registerResponder(url string, body string) {
+func registerResponder(url, body string) {
```

Comment:
Yeah open another PR for anything that's specifically cleanup related so we can merge that no question